src/api/Intelligence.js

```diff
@@ -17,10 +17,10 @@ class Intelligence extends Base {
     /**
      * API endpoint to ask ai a question
      *
-     * @param question
+     * @param {QuestionType} question - Object should at least contain the prompt, which is the question to ask
      * @param {Array<object>} items - Array of items to ask about
-     * @param dialogueHistory
-     * @param options
+     * @param {Array<QuestionType>} dialogueHistory - Array of previous questions object that already have answers
```

Comment:
naming `QuestionType` with a `-Type` suffix instead of `Question` is like naming something `FooInterface` instead of `Foo`, right? is that an existing naming convention in this repo? if not, can we rename `QuestionType` to `Question`?

is it possible to make these types stronger, eg. define a `QuestionWithAnswer` type that is `Question & { answer... }`

Reply:
that type is coming from the shared-feature, its the name they are using